### DISCUSSION TITLE
.config/far2l/NetRocks/sites.cfg

### DISCUSSION THREAD

**Discussion by verasminsk** [elfmz/far2l#945] [Q&A] [answered]
Подскажите куда пропали настройки?
![image](https://user-images.githubusercontent.com/18266668/109474786-ebc92880-7a85-11eb-841f-22f1963e1ef5.png)

файл настроек есть ".config/far2l/NetRocks/sites.cfg" и не пустой :)

**Comment by verasminsk** [comment, marked as answer]
**solution to the problem**
_~/.config/far2l/NetRocks/sites.cfg_
нужно скопировать в
_~/.config/far2l/plugins/NetRocks/sites.cfg_

**Comment by elfmz** [comment]
Хм, он должен был быть сам скопироваться при апгрейде. Что там у вас в конце ~/.cache/far2l/upgrade.log (или ~/.config/far2l/upgrade.log) написалось?

**Comment by verasminsk** [comment]
Upgrade_MoveFile('/home/veras/.config/far2l/bookmarks.ini', '/home/veras/.config/far2l/settings/bookmarks.ini') - ERROR=2
Upgrade_MoveFile('/home/veras/.config/far2l/plugins.ini', '/home/veras/.config/far2l/plugins/state.ini') - DONE
Upgrade_MoveFile('/home/veras/.config/far2l/viewer.pos', '/home/veras/.config/far2l/history/viewer.pos') - DONE
Upgrade_MoveFile('/home/veras/.config/far2l/editor.pos', '/home/veras/.config/far2l/history/editor.pos') - DONE
CheckForConfigUpgrade: ERROR=256 CMD='mv -f "/home/veras/.config/far2l/NetRocks" "/home/veras/.config/far2l/plugins/"'
---- Upgrade finished on Mon Mar  1 11:56:03 2021

**Comment by elfmz** [comment]
хм,надо было все таки сделать перенаправление stderr этой mv в лог..

**Comment by elfmz** [comment]
Подумал, полагаю дело было так: вы проинсталлили новую версию не закрывая инстансы старой, новый нетрокс подгрузился в старую версю сделал там новый файлик и к моменту запуска таки новой версии mv уже не смогла перенести туда настройки.
Чтож, костылек на такой случай уже заточен и скоро будет вмержен мастер.

**Reply by verasminsk** [reply]
Я думаю что те кто пользуются фаром вряд ли будут его закрывать и запускать терминал что бы обновить систему например командой которая живет в хистори фара =)
 _sudo apt update && sudo apt -y upgrade && sudo apt -y full-upgrade && sudo apt -y autoremove_

**Reply by elfmz** [reply]
Ну теперь костылек вмержен и должно норм переносить настройки и в таком состоянии. Впрочем для тех кто уже проапдейтился он уже не поможет и надо тока ручками
